org.opent2t.onboarding.wifi.flux/js/thingOnboarding.js

```diff
@@ -0,0 +1,75 @@
+var q = require('q');
+var dgram = require('dgram');
+
+/**
+ * Discovers and onboard Flux WiFi bulbs
+ */
+class Onboarding {
+    constructor() {
+        this._discoveryTimeout = 10000;
+
+        // Magic for discovering a Flux WiFi bulb, reverse engineered with WireShark.
+        this._discoveryPort = 48899;
+        this._discoveryMessage = 'HF-A11ASSISTHREAD';
+    }
+
+    onboard() {
+        // Flux WiFi bulbs require no authentication.
+    }
+
+    /**
+     * Discovers any devices on the current network using a number of different methods.  Each time a device is found
+     * that is understood by opent2t, it will be onboarded and the discoveryCallback will be called with the device info
+     * 
+     */
+    discover(discoveryCallback, timeout) {
```

Comment:
who calls this callback? So this would be configured say in the bridge library yes?